EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c

```diff
@@ -19,6 +20,7 @@
 #define BATTERY_FAULTS_LENGTH (3)
 #define BATTERY_DETAILS_LENGTH (51)
 #define MPPT_DETAILS_LENGTH (13)
+#define LIGHTS_DETAILS_LENGTH (7)
```

Comment:
The message has only 3 bytes of content. While this matches the protocol I think the protocol is wrong here.